.travis.yml

```diff
@@ -123,5 +143,5 @@ after_success:
     - pip install wheelhouse-uploader
     - python -m wheelhouse_uploader upload --local-folder
```

Comment:
a silly question, don't we need to increase the version (i.e. `x.x.x.dev{N}`) before upload?

Reply:
Testing on my rackspace cloud account, `wheelhouse_uploader` does this for us with timestamping. For example the wheel's name becomes:

```
scikit_learn-0.21.dev0+20180830121050-cp37-cp37m-manylinux1_x86_64.whl
```